introduction.md

```diff
@@ -1,2 +1,20 @@
 Rage has entered the dojo.
 JavaScript
+*Git
+*JavaScript
+*HTML & CSS
+*C++
+*Embedded Systems
+*WarMachine
+*Snowboarding
+*DrumSet
+*Guitar
+*Cleaning
+*CrossFit
+*Cooking
+*Driving
+*Learning
+*WorkEthic
+*FunEthic
+*Hockey
+*BasketBall
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?